jdi-bdd-tests/src/main/java/io/github/com/custom/CustomMenu.java

```diff
@@ -1,9 +1,8 @@
 package io.github.com.custom;
 
-import com.epam.jdi.light.elements.complex.Menu;
 import com.epam.jdi.light.elements.complex.WebList;
 
-public class CustomMenu extends Menu {
+public class CustomMenu extends WebList {
```

Comment:
this is necessary?

Reply:
Changes from master, supposed to be necessary